mission_control/navigator_missions/nodes/move_command

```diff
@@ -55,9 +55,12 @@ def main(args):
         res = yield n.move.circle_point(target_point, direction).go()
 
     else:
-        movement = getattr(n.move, args.command)
+        shorthand = {"f":"forward", "b":"backward", "l":"left", "r":"right", "yl":"yaw_left", "yr":"yaw_right"}
+        command = args.command if args.command not in shorthand.keys() else shorthand[args.command]
+        movement = getattr(n.move, command)
 
-        unit = 'm'
+        trans_move = command[:3] != "yaw"
+        unit = "m" if trans_move else "rad"
```

Comment:
No, these are the std units for translations or rotations, your code below still handles non-std units.